### PR TITLE
Gstreamer 1.22.6 => 1.22.8

### PR DESCRIPTION
**Pull request by uberhacker**

### CODE REVIEW

File: packages/gstreamer.rb [hunk 1]
```diff
@@ -1,24 +1,25 @@
-require 'package'
+require 'buildsystems/meson'
 
-class Gstreamer < Package
+class Gstreamer < Meson
   description 'GStreamer is a library for constructing graphs of media-handling components.'
   homepage 'https://gstreamer.freedesktop.org/'
-  @_ver = '1.22.6'
-  version "#{@_ver}-1"
+  @_ver = '1.22.8'
```
Comment (Zopolis4) [inline packages/gstreamer.rb:6]: Do we still need to be using `@_ver` here?

Comment (uberhacker) [reply packages/gstreamer.rb:6]: It will be needed for `"#{@_ver}-1"` again without affecting the original version.

Comment (Zopolis4) [reply packages/gstreamer.rb]: Do we anticipate the need for #{@_ver}-1 in the future? Most packages don't use @_ver by default, and we only use it when we need it.

Comment (Zopolis4) [reply packages/gstreamer.rb]: Never mind, I didn't see the updated changes.

**Review by Zopolis4 — APPROVED** [review]

**Pull request merged**